docs/recipes/babelrc.md

```diff
@@ -30,6 +30,21 @@ You can override the default Babel configuration AVA uses for test transpilation
 }
 ```
 
+## Provide ES2015 Polyfills to your tests
```

Comment:
Thank you @novemberborn - that's a absolute valid point. I will change that.
- [ ] Remove ES2015-reference